### PR TITLE
React core integration

### PR DESCRIPTION
**Pull request by alebinson**
Map-Colonies React core package integrated instead of MUI react components package

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -16,11 +16,12 @@ const store = rootStore.create(
...
 );
 ReactDOM.render(
-  <React.StrictMode>
+  // <React.StrictMode>
     <StoreProvider value={store}>
       <App />
     </StoreProvider>
-  </React.StrictMode>,
+  // </React.StrictMode>
+  ,
```
Comment (CptSchnitz) [inline src/index.tsx:24]: is it still not possible to return it?

**Comment by syncush** [comment]
This PR closes #12

File: src/index.tsx (hunk 1 above)
Comment (alebinson) [reply src/index.tsx:24]: Sorry, but not. Ripple core component uses ReactDOM.findDOMNode() which is deprecated 
I open a backlog issue https://github.com/MapColonies/shared-components/issues/27

File: src/App.tsx [hunk 2]
```diff
@@ -1,28 +1,35 @@
...
+import { ThemeProvider as RMWCThemeProvider, RMWCProvider, Themes } from '@map-colonies/react-core';
+import { CssBaseline } from '@map-colonies/react-components';
+import { useMediaQuery } from '@map-colonies/react-components';
+import '@map-colonies/react-core/dist/theme/styles';
+import '@map-colonies/react-core/dist/button/styles';
+import '@map-colonies/react-core/dist/tooltip/styles';
+import '@map-colonies/react-core/dist/menu/styles';
+import '@map-colonies/react-core/dist/select/styles';
+import '@map-colonies/react-core/dist/circular-progress/styles';
+import '@map-colonies/react-core/dist/typography/styles';
```
[the remark this answers is not in the document]
Comment (alebinson) [reply src/App.tsx:14]: Let's run for a while in this way.
It's by design, also may give a flexibility in styling customization.

File: src/conflicts/components/has-resolved-filter.tsx [hunk 3]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
-import { Select, MenuItem } from '@material-ui/core';
 import { observer } from 'mobx-react-lite';
+
```
Comment (syncush) [inline src/conflicts/components/has-resolved-filter.tsx:3]: redundant line

File: src/conflicts/components/has-resolved-filter.tsx [hunk 4]
```diff
@@ -1,6 +1,8 @@
...
+import { Select } from '@map-colonies/react-core';
+
```
Comment (syncush) [inline src/conflicts/components/has-resolved-filter.tsx:5]: redundant line

File: src/index.tsx [hunk 5]
```diff
@@ -20,7 +20,8 @@ ReactDOM.render(
     <StoreProvider value={store}>
       <App />
     </StoreProvider>
-  </React.StrictMode>,
+  </React.StrictMode>
+  ,
```
Comment (syncush) [inline src/index.tsx:24]: redundant line, `,` should be on the same line

File: src/conflicts/components/has-resolved-filter.tsx (hunk 3 above)
Comment (alebinson) [reply src/conflicts/components/has-resolved-filter.tsx:3]: Done

File: src/conflicts/components/has-resolved-filter.tsx (hunk 4 above)
Comment (alebinson) [reply src/conflicts/components/has-resolved-filter.tsx:5]: Done

File: src/index.tsx (hunk 5 above)
Comment (alebinson) [reply src/index.tsx:24]: Done

**Review by syncush — APPROVED** [review]

**Pull request merged**